met-web/src/components/engagement/view/SurveyBlock.tsx

```diff
@@ -33,7 +33,7 @@ const SurveyBlock = ({ startSurvey }: SurveyBlockProps) => {
             <Grid container direction="row" alignItems="flex-end" justifyContent="flex-end" spacing={2}>
                 <Grid item xs={12}>
                     <>
-                        <If condition={status_text !== undefined}>
+                        <If condition={status_text !== ''}>
```

Comment:
Maybe try condition={!!status_text} or condition={Boolean(status_text)}. '', undefined, null, 0 are falsey values

Reply:
updated it, thanks.